.github/workflows/gcc.yml

```diff
@@ -45,4 +45,4 @@ jobs:
           CFLAGS: "-std=${{ matrix.c }}"
           # TODO: -pedantic-errors here won't compile
           CXXFLAGS: "-std=${{ matrix.cpp }}"
-        run: .github/workflows/build_ubuntu-20.04.sh $HOME/install
+        run: .github/workflows/build_ubuntu-20.04.sh $HOME/install -Werror
```

Comment:
The CFLAGS right above is not working for you? That's what is determining the standard used, so it better works.

Reply:
> The CFLAGS right above is not working for you? That's what is determining the standard used, so it better works.

No, it's not working locally, nor on the CIs.
My take is that the configure file is based on a too old version of autoconf (v2.13 from 1999). See `config.log` locally to see there are a number of warnings, which ends up as errors during configuration with `-Werror` causing configuration failure. 
For a permanent solution configure.in should be updated. While this only affects certain CI build for testing purposes I'd say this workaround is acceptable for the time being.

Reply:
Then just add a note to the definition of `makecmd` that it can be removed once configure works with -Werror enabled.